pkg/importer/dataStream.go

```diff
@@ -111,6 +113,8 @@ type DataStreamOptions struct {
 	InsecureTLS bool
 	// ScratchDataDir is the path to the scratch space inside the container.
 	ScratchDataDir string
+	// VolumeMode indicates whether the PV we import the disk image to, is a block PV or not
+	VolumeMode string
```

Comment:
Why not make this an actual PersistentVolumeMode instead of a string, that way you don't have to cast everything to a string all over the place.

Reply:
Sure.

Reply:
By the way, I followed the ContentType  which has the same casting issue

Reply:
Yes that is wrong as well, I already addressed it in the refactor PR. If we have actual types available (even if they just translate to strings), we want to use the actual types.